packages/frontend/component/src/ui/slider/index.css.ts

```diff
@@ -0,0 +1,54 @@
+import { cssVarV2 } from '@toeverything/theme/v2';
+import { style } from '@vanilla-extract/css';
+
+export const sliderContainerStyle = style({
+  position: 'relative',
+  width: '250px',
```

Comment:
- This property is invalid since it's `inline` element
- We can't customize `width` outside